libraries/WiFi/src/WiFiMulti.cpp

```diff
@@ -160,6 +161,8 @@ uint8_t WiFiMulti::run(uint32_t connectTimeout)
             log_i("[WIFI] Connecting BSSID: %02X:%02X:%02X:%02X:%02X:%02X SSID: %s Channel: %d (%d)", bestBSSID[0], bestBSSID[1], bestBSSID[2], bestBSSID[3], bestBSSID[4], bestBSSID[5], bestNetwork.ssid, bestChannel, bestNetworkDb);
 
             WiFi.begin(bestNetwork.ssid, bestNetwork.passphrase, bestChannel, bestBSSID);
+            if (ipv6_support == true)
+                WiFi.IPv6(true);
```

Comment:
can't you just proxy `WiFiMulti::IPv6` to `WiFi.IPv6` and not need to keep a redundant variable?
```cpp
void WiFiMulti::IPv6(bool state) {
     WiFi.IPv6(state);
}
```

Reply:
Will check

Reply:
I think the main reason is that when calling `WiFiMulti::IPv6()`, the `WiFi` object is not yet available. The whole creation of `WiFi` must probably be done in the right sequence, and `WiFiMulti` needs to keep track if it enables IPv6 or not.

Reply:
The `WiFi` object is available before even `setup()` is called and the user has a chance to call `WiFi.IPv6()` so it should be OK. Can you please confirm?

Reply:
This is fixed now, as you suggested, and tested: confirmed to work fine.